src/database/migrations.py

```
@@ -8,12 +8,13 @@
 from alembic import command
 from alembic.config import Config
 from alembic.script import ScriptDirectory
-from sqlalchemy import inspect
+from sqlalchemy import create_engine, inspect, text
+from sqlalchemy.engine.url import make_url
 from sqlalchemy.exc import SQLAlchemyError
 
 from sqlalchemy.ext.asyncio import AsyncConnection
```

Flake8 (7.2.0)

[error] 'sqlalchemy.ext.asyncio.AsyncConnection' imported but unused

(F401)